website/docs/r/event_orchestration_service.html.markdown

```diff
@@ -0,0 +1,202 @@
+        }
+        extractions {
+          # Demonstrating a template-style extraction
+          template = "{{variables.hostname}}"
+          target = "event.custom_details.hostname"
+        }
+        extractions {
+          # Demonstrating a regex-style extractions
+          source = "event.source"
+          regex = "www (.*) service"
+          target = "event.source"
+        }
+        # Id of the next set
+        route_to = "step-two"
+      }
+    }
+  }
+  sets {
+    id = "step-two"
+    rules {
+      label = "All critical alerts should be treated as P1 incident"
+      conditions {
+        expression = "event.severity matches 'critical'"
+      }
+      actions {
+        annotate = "Please use our P1 runbook: https://docs.test/p1-runbook"
+        priority = "P0IN2KQ"
+      }
+    }
+    rules {
+      label = "If there's something wrong on the canary let the team know about it in our deployments Slack channel"
+      conditions {
+        expression = "event.custom_details.hostname matches part 'canary'"
+      }
+      # create webhook action with parameters and headers
+      actions {
+        automation_actions {
+          name = "Canary Slack Notification"
+          url = "https://our-slack-listerner.test/canary-notification"
+          auto_send = true
+          parameters {
+            key = "channel"
+            value = "#my-team-channel"
+          }
+          parameters {
+            key = "message"
+            value = "something is wrong with the canary deployment"
+          }
+          headers {
+            key = "X-Notification-Source"
+            value = "PagerDuty Incident Webhook"
+          }
+        }
+      }
+    }
+    rules {
+      label = "Never bother the on-call for info-level events outside of work hours"
+      conditions {
+        expression = "event.severity matches 'info' and not (now in Mon,Tue,Wed,Thu,Fri 09:00:00 to 17:00:00 America/Los_Angeles)"
+      }
+      actions {
+        suppress = true
+      }
+    }
+  }
+  catch_all {
+    actions { }
+  }
+}
+```
+## Argument Reference
+
+The following arguments are supported:
+
+* `service` - (Required) ID of the Service to which this Service Orchestration belongs to.
+* `sets` - (Required) A Service Orchestration must contain at least a "start" set, but can contain any number of additional sets that are routed to by other rules to form a directional graph.
+* `catch_all` - (Required) When none of the rules match an event, the event will be routed according to the catch_all settings.
+
+### Sets (`sets`) supports the following:
+* `id` - (Required) The ID of this set of rules. Rules in other sets can route events into this set using the rule's `route_to` property.
+* `rules` - (Optional) The service orchestration evaluates Events against these Rules, one at a time, and routes each Event based on the first rule that matches. If no rules are provided as part of Terraform configuration, the API returns empty list of rules.
```

Comment:
**Blocker:** This isn't a Router so the phrase "routes each event" doesn't seem quite right to me. Maybe this instead?

* `rules` - (Optional) The Service Orchestration evaluates Events against these Rules, one at a time, and applies all the actions for first rule it finds where the event matches the rule's conditions. If no rules are provided as part of Terraform configuration, the API returns empty list of rules.